extension/src/time_vector.rs

```diff
@@ -115,6 +119,55 @@ pub fn unnest<'a>(
     )
 }
 
+/// Util function to convert from *const ::std::os::raw::c_char to String
+/// TimestampTz -> *const c_char -> &CStr -> &str -> String
+pub fn timestamptz_to_string(time: pg_sys::TimestampTz) -> Result<String, Utf8Error> {
+    let char_ptr = unsafe { timestamptz_to_str(time) };
+    let c_str = unsafe { CStr::from_ptr(char_ptr) };
+    c_str.to_str().map(|s| s.to_owned())
+}
+
+#[pg_extern(immutable, schema = "toolkit_experimental", parallel_safe)]
+pub fn format_timevector<'a>(series: Timevector_TSTZ_F64<'a>, format_string: String) -> String {
```

Comment:
I don't love the name here.  I would prefer something like `to_json`, though I realize that might be a stretch if we're not validating the result is proper JSON.  `to_string`, `to_text`, `output_as`?

Reply:
Thoughts on using the following for the functions we have / will have after I address other comments?:
- to_text() for the base function
- to_plotly() for plotly specific output
- possibly to_json() for any output that we want to actually parse to make sure its valid json in the future (we do not currently do this)

Reply:
This seems like a good set of functionality for initial release.